roles/oawaiver/molecule/default/converge.yml

```diff
@@ -0,0 +1,22 @@
+---
+- name: Converge
+  hosts: all
+  vars:
+    - running_on_server: false
+    - nodejs__upstream_release: 'node_12.x'
+    - nodejs__upstream_key_id: '68576280'
+    - mariadb__server: 'localhost'
+    - mariadb__client: 'localhost'
+  become: true
+  pre_tasks:
+    - name: update cache
+      apt:
+        update_cache: true
+        cache_valid_time: 600
+  tasks:
+    - name: "Include mariadbserver"
+      include_role:
+        name: "mariadbserver"
```

Comment:
This was needed in order to ensure that the creation of MariaDB users and databases are properly tested. Unfortunately, I am still struggling to resolve one error with the Molecule test suite.